result/convergence.yaml

```diff
@@ -0,0 +1,102 @@
+            type: string
+          x-constraint:
+          - "/components/schemas/Flow/properties/name"
+          x-field-uid: 1
+    Convergence.Metric:
+      description: |-
+        The container for convergence measurements
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a flow. 
+          type: string
+          x-field-uid: 1
+        data_plane_convergence_us:
+          description: |-
+            The convergence time measured from the data plane perspective only. 
+            It measures the time from BTT, when the rate on Test Port 2 crosses 
+            below the Rx Threshold until an acceptable amount of traffic was 
+            received at time ATT, when the rate crosses above the configured 
+            rx_rate_threshold.
+          type: number
+          format: double
+          x-field-uid: 2
+        control_plane_data_plane_convergence_us:
+          description: |-
+            The total convergence time, between the event that caused the
+            switchover (at tEvent) until an acceptable amount of traffic was 
+            received at time ATT, when the rate crosses above the configured 
+            rx_rate_threshold.
+          type: number
+          format: double
+          x-field-uid: 3
+        events:
+          description: |-
+            The events that were used to determine the convergence analytics.
+          type: array
+          items:
+            $ref: '#/components/schemas/Convergence.Event'
+          x-field-uid: 4
+    Convergence.Event:
+      description: |-
+        A container for an event that has occurred in the system.
+      type: object
+      properties:
+        type:
+          description: |-
+            The type of event that occurred.
+          type: string
+          example: route_withdraw
+          x-field-uid: 1
+          x-enum:
+            link_down:
+              x-field-uid: 1
+            link_up:
+              x-field-uid: 2
+            route_withdraw:
+              x-field-uid: 3
+            route_advertise:
+              x-field-uid: 4
+            flow_rx_rate_above_threshold:
+              x-field-uid: 5
+            flow_rx_rate_below_threshold:
+              x-field-uid: 6
+        source:
+          description: |-
+            The source of the event. 
+            The source MUST be the value of one of the x-constraint paths,
+            which means the source MUST be a unique name in the configuration. 
+          type: string
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
+          - '/components/schemas/Flow/properties/name'
+          - '/components/schemas/Device.Bgpv4Route/properties/name'
+          - '/components/schemas/Device.Bgpv6Route/properties/name'
+          example: Bgpv4 Route Range 1
+          x-field-uid: 2
+        begin_timestamp_ns:
+          description: |-
+            The time in nanoseconds at which the event began
```

Comment:
since what .. start of traffic ? [ note : pause/resume ]

Reply:
Please provide a suggestion after experimentation as to what the impl. should base the start of time calculation on e.g. start traffic or last start/resume traffic.